com.core447.StreamController.yml

```diff
@@ -0,0 +1,105 @@
+id: com.core447.StreamController
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: launch.sh
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=xdg-documents
+  - --filesystem=home
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=session-bus # Allow plugins to connect to the session bus
```

Comment:
why the session bus? this is not allowed by default.

Reply:
To allow the plugins to integrate with desktops app. This can be used to request the installation of Gnome extensions, integrate with the Gnome calendar and stuff like that.

Reply:
You should rather add the interfaces individually.

Reply:
ok, did it in 4367aa9dffbfa2a367846869d6fca24416f89443